sass/_valkey.scss

```diff
@@ -530,14 +530,25 @@ pre table {
   border-collapse: collapse;
 }
 
-.command-entry {
-  border-bottom: 1px dotted $line;
-  margin-top: 0.5rem;
-  margin-bottom: 0.5rem;
-  padding-top: 1rem;
-  padding-bottom: 1rem;
+.command-group {
+  .command-entry {
+    border-bottom: 1px dotted $line;
```

Comment:
I've never seen nested CSS blocks like this. Is it a SCSS specific or some new CSS feature? (I used to know CSS well, a decade ago)

Reply:
...both! The site preprocess SCSS into plain, old CSS but [nested CSS is a thing now](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_nesting/Using_CSS_nesting).

Reply:
Hehe, I'm getting old. I should stick to C. It's changing too, but less. :D